README.rst

```diff
@@ -21,18 +18,124 @@ Page <http://ait-core.readthedocs.io/en/latest/project_setup.html>`__
+==================
 
 The project's `User and Developer Mailing List <https://groups.google.com/forum/#!forum/ait-dev>`__ is the best way to communicate with the team, ask questions, brainstorm plans for future changes, and help contribute to the project.
 
 This project exists thanks to the dedicated users, contributors, committers, and project management committee members. If you'd like to learn more about how the project is organized and how to become a part of the team please check out the `Project Structure and Governance <https://github.com/NASA-AMMOS/AIT-Core/wiki/Project-Structure-and-Governance>`__ documentation.
 
 Contributing
-------------
+============
+
+Thank you for your interest in contributing to AIT! We welcome contributions from people of all backgrounds and disciplines. While much of the focus of our project is software, we believe that many of the most critical contributions come in the form of documentation improvements, asset generation, user testing and feedback, and community involvement. So if you're interested and want to help out please don't hesitate! Send us an email on the public mailing list below, introduce yourself, and join the community.
+
+Communication
+-------------
+All project communication happens via mailing lists. Discussions related to development should happen on the public `Developer and User Mailing List <https://groups.google.com/forum/#!forum/ait-dev>`__. If you're new to the community make sure to introduce yourself as well!
+
+Dev Installation
+----------------
+As always, we encourage you to install AIT into a virtual environment of your choosing when you set up your development environment. AIT uses `poetry` for package management. As such, you can
+Before setting up your development environment you will need the following installed and ready to use:
+
+- A virtual environment "manager" of your choosing with a configured and activated virtual environment. Since AIT uses `poetry` you can consider leveraging its `environment management <https://python-poetry.org/docs/managing-environments/>`__ functionality as well.
+- `pyenv <https://github.com/pyenv/pyenv>`__ so you can easily install different Python versions
+- `poetry <https://python-poetry.org/docs/#installation>`__ installed either to your specific virtual environment or system-wide, whichever you prefer.
+
+Install the package in "editable" mode with all the development dependencies by running the following::
+
+    poetry install
+
+As with a normal installation you will need to point `AIT_CONFIG` at the primary configuration file. You should consider saving this in your shell RC file or your virtual environment configuration files so you don't have to reset with every new shell::
+
+    export AIT_CONFIG=/path/to/ait-core/config/config.yaml
+
+Finally, you should configure `pre-commit` by running the following. This will install our pre-commit and pre-push hooks::
+
+    pre-commit install && pre-commit install -t pre-push
```

Comment:
Is pre-commit a separate tool that needs its own installation? If so it should be included in the tool list above.

Reply:
Or maybe I just needed to re-start my virtual env?